eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/SiteAccessMatcherRegistryPass.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * File containing the ChainConfigResolverPass class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Compiler;
+
+use eZ\Bundle\EzPublishCoreBundle\SiteAccess\SiteAccessMatcherRegistry;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * The SiteAccessMatcherRegistryPass will register all services tagged as "ezpublish.matcher.siteaccess" to the registry.
+ */
+final class SiteAccessMatcherRegistryPass implements CompilerPassInterface
+{
+    public const MATCHER_TAG = 'ezpublish.matcher.siteaccess';
```

Comment:
WDYT about renaming tag name to 

```suggestion
    public const MATCHER_TAG = 'ezplatform.siteaccess.matcher';
```

?

IMHO  it sounds more natural and is easier to remember than `ezplatform.matcher.siteaccess`. Also, it will allow us to group all potential DIC tags related to siteaccesses with  `ezplaform.siteaccess` prefix (same as we group all field type related tags using `ezplatform.field_type.value_mapper` prefix) .